src/App.scss

```diff
@@ -1,7 +1,5 @@
 // Importing Global Variables
-@import '~@redhat-cloud-services/frontend-components-notifications/index.css';
-@import '~@redhat-cloud-services/frontend-components/index.css';
-@import '~@redhat-cloud-services/frontend-components-utilities/files/Utilities/_variables';
+@import '~@redhat-cloud-services/frontend-components-utilities/Utilities/_variables';
```

Comment:
The sass files location has also changed. See: https://github.com/RedHatInsights/frontend-components/tree/master/packages/utils#scss-files